configure.ac

```
@@ -1,7 +1,7 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-AC_INIT([libteam], [1.31-wmo3], [jiri@resnulli.us])
+AC_INIT([libteam], [1.31-wmo4], [jiri@resnulli.us])
```

```suggestion
AC_INIT([libteam], [1.31-wmo4], [andy.koszela@westermo.com])
```

We should probably not use the original upstream email here. Feel free to use your own instead of mine.

---

Fixed